### DISCUSSION TITLE
Crowdstrike logs shipping to Security Onion

### DISCUSSION THREAD

**Discussion by Blason** [Security-Onion-Solutions/securityonion#5576] [Unsupported Versions] [answered]
Hi Team,

Wondering if anyone has succeeded shipping crowdstrike logs to SO?
Any thig other that I needed to do?

TIA
Blason R

**Comment by cyberkimh** [comment, marked as answer]
Yes, there's a write up here: https://stephen-ridgway.medium.com/security-onion-crowdstrike-connector-installation-c6c5d77f4096
